Blaise-Shyaka/Mentors
# [#171408793] Create signup API endpoint

# What does this PR do?

It creates a signup API endpoint for users

# Description of task to be completed

- Write tests for the `POST /auth/signup endpoint`
- Create a route for the `POST /auth/signup endpoint`
- Initiate Travis CI build
- Get coverage badges from coveralls
-  host the app on Heroku

# How should this be manually tested?

- Clone this repository
- In the terminal run `npm init` to install all the dependencies
- Run `npm test` command
- 1 test should pass

# What are the relevant pivotal tracker stories?

#171408793
